### DISCUSSION TITLE
Metdata processor value error

### DISCUSSION THREAD

**Discussion by hhh321747** [UMEP-dev/UMEP#646] [Q&A]
Ldown-beyond what is expected at line：[1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23]
![Uploading cbbcea0e8e429edda94c36ebf8a7188.png…]()

**Comment by biglimp** [comment]
Can you share your meteorological data please.

**Reply by hhh321747** [reply]
![cbbcea0e8e429edda94c36ebf8a7188](https://github.com/user-attachments/assets/aaa478a6-45cb-4018-8fd6-1ee9b159eea6)

**Reply by biglimp** [reply]
You use llr as Ldown which is unreasonable. Incoming longwave radiation cannot be negative and should have values within the range of appr. 100-500 Wm-2.

**Reply by hhh321747** [reply]
![68b83a7882cd1bf24aca1c6e7f8ec35](https://github.com/user-attachments/assets/b93b2072-043e-4b3e-ad60-bec1676b4c0e)
Now the question arises again. Is it 24 hours? Is it 0-23 or is it 1-23 plus 0 o 'clock the next day? Thanks a million

**Reply by biglimp** [reply]
0-23. Something else must be wrong, e.g. what columns you use to specify hour etc.
